test/FilterMethodsTest.py

```diff
@@ -0,0 +1,83 @@
+'''
+Created on 21.03.2018
+
+@author: gregor
+'''
+import unittest
+from ipet.evaluation import IPETFilter
+from _ast import Attribute
+from ipet.evaluation.IPETFilter import IPETValue
+
+
+class FilterMethodsTest(unittest.TestCase):
+
+    def testFilterComparison(self):
+
+        # test if different operators are correctly distinguished
+        operatorlist = ["le", "ge", "lt", "gt", "eq", "neq"]
+        for operator1 in operatorlist:
+            for operator2 in operatorlist:
+                f1 = IPETFilter("A", "B", operator1)
+                f2 = IPETFilter("A", "B", operator2)
+                eq = f1.equals(f2)
+                self.assertEqual(eq, operator1 == operator2, "Bla")
```

Comment:
Yay for testing, but may I suggest that we use a more meaningful errormessage?

Reply:
Yes, I will change this. Remind me to show you a best of of your meaningful error messages at some point in time.